docs/Products/VerifiedAssetOracle/vaoDashboard.md

```diff
@@ -0,0 +1,127 @@
+---
+sidebar_position: 2
+description: Verified Asset Oracle Dashboard
+keywords: [dashboard, Verified Asset Oracle, VAO, RWA Oracle Data]
+---
+ # Verified Asset Oracle Dashboard
+
+The Verified Asset Oracle Dashboard enables users to visualize data verified by [Chronicle's Verified Asset Oracle](/Products/VerifiedAssetOracle/verifiedAssetOracle.md). It provides insights into key financial metrics associated with verified assets such as Net Asset Value per Share (NAV/S), 7-day average annualized yield, custodian information, and more.
+
+This dashboard is designed to enhance transparency and trust in onchain representations of real-world assets by offering a clear visualization of their value and underlying assets.
+
+# How is the Chronicle Dashboard different from other RWA visual representations?
+Chronicle provides end-to-end data about a tokenized fund by connecting directly to the custodian. Users can verify attestation signatures in the browser via the `Verify` button.
+
+The signature process has two layers:
+
+ **1. Publishing to IPFS:** Data is fetched from the custodian, formatted, and published to IPFS. The IPFS URL is content-based, so any change in the data alters the URL, helping detect potential manipulation.
+
+Data is pulled from the IPFS document stored which can be accessed using a unique [CID (Content Identifier).](https://docs.ipfs.tech/concepts/content-addressing#) The CID itself is a hash of the document, meaning that if the content changes even slightly, the CID will change. This acts like a built-in verification layer, if the CID matches, the content matches as well. There is also a **`?checksum=`** parameter in the URL, which is a hash of the document’s actual contents.
+
+`e.g.: ipfs://QmPv8UFfZAk7CiNLAW16ZBic741aWtFMXjbU3KkxrRR8Ma?checksum=0xd3e407f7243336f4ba79027b439bcf6a534ffed8fb4d1cc3acfe014d28127774`
+
+The **CID ensures the data hasn’t been tampered with**, as it won’t resolve if the content changes.
+
+**2. Checksum verification:** A checksum of the data is added and verified on-chain via a verifier contract.
+
+The checksum proves that the content of the document has remained the same since its initial publication.
+
+### Verification Flow
+
+When someone downloads the document from the link:
+
+- They can calculate the hash of the content themselves.
+- Then they compare it to the `?checksum=` in the URL (which was previously published).
+- If it matches, it proves that the document is the same one that was originally published
+
+## Blockchains
+
+This module shows the contract addresses of all oracles deployed for the asset. These contracts are responsible for reporting the price of the asset on the chain where they are deployed.
+
+
+<div style={{textAlign: 'center'}}>
+<img
+    src="  ../../img/Products/VAO/blockchains.png"
+    alt="Blockchains card Illustration"
+/>
+</div>
+
+## Administration
```

Comment:
Heads up: The Administration info is only available from certain funds and not others, so it will not be a universal feature of the Dashboard